apps/settings/js/carrier.js

```diff
@@ -119,6 +119,9 @@ var CarrierSettings = (function() {
       });
     }
 
+    if (!_mobileConnections[0].iccId) {
```

Comment:
I've tested in flame. But yes, we need do handle multiple sim cards case here. Thanks.